README.md

```diff
@@ -31,66 +31,89 @@ The Perl container is set up so that Apache will load additionally provided .con
-`oc process -f openshift/templates/dancer-mysql.json | oc create -f - `
-3. Start the build  
-`oc start-build dancer-app`
+3. Clone your repository to your development machine
+2. Add a Perl application from the dancer-mysql template and specify the source url to be your forked repo  
+
+		$ oc process -f openshift/templates/dancer-mysql.json -v SOURCE_REPOSITORY_URL=<your repository location> | oc create -f - 
+
 4. Watch your build progress  
-`oc build-logs -f dancer-app-1`  
+
+		$ oc build-logs dancer-app-1
+
 5. Wait for frontend and database pods to be started (this can take a few minutes):  
-`oc get pods`  
-Sample output:  
->POD                  IP            CONTAINER(S)   IMAGE(S)                                                                                                             HOST                                            LABELS                                                                              STATUS       CREATED      MESSAGE
-dancer-app-1-build                                                                                                                                                     ip-10-230-142-143.ec2.internal/10.230.142.143   build=dancer-app-1,buildconfig=dancer-app,name=dancer-app,template=dancer-example   Succeeded    15 minutes   
-                                   sti-build      openshift/origin-sti-builder:v0.5.3                                                                                                                                                                                                                      Terminated   15 minutes   exit code 0
-frontend-1-w6cef     172.17.0.50                                                                                                                                       ip-10-230-142-143.ec2.internal/10.230.142.143   deployment=frontend-1,deploymentconfig=frontend,name=frontend                       Running      3 minutes    
-                                   dancer-app     172.30.221.178:5000/demo/origin-dancer-app@sha256:dd8452a64d8cf9ba36a461603306ae440b289f819c77dae1989c8980f3243f8e                                                                                                                                       Running      3 minutes    
-mysql-1-4edf0        172.17.0.45                                                                                                                                       ip-10-230-142-143.ec2.internal/10.230.142.143   deployment=mysql-1,deploymentconfig=mysql,name=mysql                                Running      3 minutes   
-                                   mysql          openshift/mysql-55-centos7:latest                                                                                                                                                                                                                        Running      3 minutes  
+
+		$ oc get pods -w
```

Comment:
too much repetition in this whole DB section.  it's basically the same steps as the non-DB case, we don't need to document it all twice.